### PR TITLE
Fix IE issue in animation tab

### PR DESCRIPTION
**Pull request by caleybrock**
Animations weren't saving in IE. Turns out IE doesn't infer "Content-type" in the header, but on the server side, we rely on "Content-type" being "image/png" so we're explicitly setting it. This doesn't cause any regressions in Chrome, since it's already set.

cc @Bjvanminnen

### CODE REVIEW

**Comment by caleybrock** [comment]
Updated a test to catch this case - thanks for the sinon tip 👍

File: apps/test/unit/gamelab/animationListModuleTest.js [hunk 1]
```diff
@@ -563,4 +564,34 @@ describe('animationListModule', function () {
       expect(store.getState().animationList.pendingFrames.props).to.deep.equal(libraryAnimProps);
     });
   });
+
+  describe('action: save animation', function () {
+    beforeEach(function () {
+      this.xhr = sinon.useFakeXMLHttpRequest();
```
Comment (islemaster) [inline apps/test/unit/gamelab/animationListModuleTest.js]: It looks like you never call `this.xhr.restore()`.

File: apps/test/unit/gamelab/animationListModuleTest.js [hunk 2]
```diff
@@ -563,4 +564,34 @@ describe('animationListModule', function () {
...
+      let requests = this.requests = [];
+      this.xhr.onCreate = function (xhr) {
```
Comment (islemaster) [inline apps/test/unit/gamelab/animationListModuleTest.js]: I personally prefer not using `this` in tests - I'm a little fuzzy what it's bound to.  Instead, declare locals in the `describe` block.

```javascript
describe('action: save animation', function () {
  let xhr, requests;
  beforeEach(function () {
    xhr = sinon.useFakeXMLHttpRequest();
    requests = [];
    xhr.onCreate = xhr => requests.push(xhr);
  });

  // Test cases...
});
```

File: apps/test/unit/gamelab/animationListModuleTest.js [hunk 3]
```diff
@@ -563,4 +564,34 @@ describe('animationListModule', function () {
...
+        name: 'animation_1',
+        sourceUrl: 'url',
+        frameSize: {x: 100, y: 100},
+        frameCount: 1,
+        looping: true,
+        frameDelay: 4,
+        version: null
+      };
+
+      let callback = sinon.spy();
```
Comment (islemaster) [inline apps/test/unit/gamelab/animationListModuleTest.js]: Unused?

File: apps/test/unit/gamelab/animationListModuleTest.js [hunk 4]
```diff
@@ -563,4 +564,34 @@ describe('animationListModule', function () {
...
+      saveAnimation('animation_1', libraryAnimProps);
+      expect(this.requests.length).to.equal(1);
```
Comment (islemaster) [inline apps/test/unit/gamelab/animationListModuleTest.js]: This is fine; you can also say `expect(requests).to.have.length(1);`

File: apps/test/unit/gamelab/animationListModuleTest.js (hunk 2 above)
Comment (caleybrock) [reply apps/test/unit/gamelab/animationListModuleTest.js]: fair, it's fuzzy to me too. I probably relied too heavily on the documentation example code here.

**Pull request merged**